### PR TITLE
CI: Set 10 min timeout for Argo workflows

### PR DESCRIPTION
**Pull request by IagoAbal**

### CODE REVIEW

File: .github/workflows/libs/util.libsonnet [hunk 1]
```diff
@@ -57,6 +57,7 @@
   trigger_argo_workflow: function(trigger_url, workflow_inputs) {
     'runs-on': 'ubuntu-22.04',
     needs: ['setup-docker-tag', 'get-sha'],
+    'timeout-minutes': 10,
```
Comment (aryx) [inline .github/workflows/libs/util.libsonnet:60]: need both things?

Comment (aryx) [reply .github/workflows/libs/util.libsonnet:60]: Maybe remove the --max-time?

**Review by aryx — APPROVED** [review]

File: .github/workflows/libs/util.libsonnet (hunk 1 above)
Comment (aryx) [reply .github/workflows/libs/util.libsonnet:60]: test plan?

Comment (IagoAbal) [reply .github/workflows/libs/util.libsonnet:60]: I was trying to make sure something worked, but it seems that nothing worked. 😓 The workflow for this PR timed out after 1 hour. I'll try to check with @minusworld later today.

**Comment by IagoAbal** [comment]
This didn't work, but Grayson will open a new PR.

**Pull request closed without merging**